### PR TITLE
feat: create dim_user_pii report

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR creates a report to get user information.

image

### WIP
- Create a macro to get the `dim_user_pii` model joined onto other reports.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by Ian2012** [comment]
@daniriosp I know this does include the email, will do in the future, but is there anything else you would need here?

File: models/base/sources.yml [hunk 1]
```diff
@@ -13,6 +13,35 @@ sources:
           - name: emission_time
           - name: event
 
+  - name: dim_user_pii
```
Comment (SoryRawyer) [inline models/base/sources.yml:16]: I think the way this is set up you're saying there's an existing table in ClickHouse called `dim_user_pii.user_pii` when really this should be another entry in `models/base/schema.yml`.

Comment (Ian2012) [reply models/base/sources.yml]: corrected

**Comment by daniriosp** [comment]
Yes, I would like to have the email too, but the rest is perfect, thank you very much.

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/aspects-dbt&entry.867055334=https://github.com/openedx/aspects-dbt/pull/58&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-03-18+21:27&entry.1917517419=2024-03-22+19:45&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**